src/Strategy.js

```diff
@@ -0,0 +1,73 @@
+//@flow
+
+import {expandStyle} from './compiler';
+import {Spec} from './ElementPropSpec';
+import {staticStylesheetManager, dynamicStylesheetManager} from './StylesheetManager';
+
+export const resolve = ({
+  defaultClassName,
+  defaultComponent,
+  ownProps,
```

Comment:
For the caller side it should reads `props` as `ownProps` makes unnecessary special case for the consumers of the API (what they have is just `props`). You could name resulted `props` — `resolvedProps` or something like this within a function.